src/lib/flex/flex-align/flex-align.ts

```diff
@@ -96,33 +83,23 @@ export class FlexAlignDirective extends BaseDirective implements OnInit, OnChang
    * For @Input changes on the current mq activation property, see onMediaQueryChanges()
    */
   ngOnChanges(changes: SimpleChanges) {
-    if (changes['align'] != null || this._mqActivation) {
-      this._updateWithValue();
-    }
-  }
-
-  /**
-   * After the initial onChanges, build an mqActivation object that bridges
-   * mql change events to onMediaQueryChange handlers
-   */
-  ngOnInit() {
-    super.ngOnInit();
-
-    this._listenForMediaQueryChanges('align', 'stretch', (changes: MediaChange) => {
-      this._updateWithValue(changes.value);
+    // TODO: figure out how custom breakpoints interact with this method
```

Comment:
Could change the inputs to be a protected class member allowing custom directives which override this directive to specify the custom breakpoints. That way this directive handles the default breakpoints and the custom directive handles the custom breakpoints, ngOnChanges won't be called for each others inputs only their own inputs.

Reply:
That's exactly what I had in mind 😄